index.css

```diff
@@ -0,0 +1,540 @@
+
+  nav .top {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: flex-end;
+    margin: 0 auto;
+    background: #3e3e3e;
+    padding: 0.2rem 1rem;
+    gap: 1rem;
+  }
+
+  nav .top a {
+    font-family: 'Lato', sans-serif;
+    color: #fff;
+  }
+
+  nav .bottom {
+    width: 90%;
+    display: flex;
+    padding: 0.5rem;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+  }
+
+  nav .bottom .nav-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-menu li:nth-child(6) {
+    padding: 0.5rem;
+    border: 5px solid #ec4327;
+  }
+
+  .nav-menu li:nth-child(6) a {
+    padding: 0.5rem;
+    color: #ec4327;
+  }
+
+  .nav-link {
+    font-family: 'Lato', sans-serif;
+    color: #868686;
+  }
+
+  .hero-wrapper h1 {
+    font-size: 2.5rem;
+    font-weight: 800;
+    line-height: 3rem;
+    background-clip: initial;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-image:
+      -webkit-linear-gradient(
+        45deg,
+        #f40001,
+        #ec5242,
+        rgb(255, 64, 0),
+        rgb(255, 123, 0),
+        #a00000,
+        #f65112,
+        #ec5242
+      );
+  }
+
+  .description {
+    width: 68%;
+    font-weight: 400;
+    font-size: 1.3rem;
+    line-height: 1.6rem;
+  }
+
+  .date {
+    font-size: 2.7rem;
+  }
+
+  .location {
+    font-size: 1.5rem;
+  }
+
+  .all-programs {
+    display: flex;
+    gap: 0.6rem;
+  }
+
+  .program-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .program-desc {
+    text-align: center;
+  }
+
+  .join-btn {
+    display: none;
+  }
+
+  .join-btn2 {
+    display: block;
+    font-family: 'Lato', sans-serif;
+    font-weight: 400;
+    font-size: 1rem;
+    background: transparent;
+    color: #fff;
+    padding: 1.25rem;
+    margin-top: 1rem;
+    text-decoration: underline;
+  }
+
+  .all-speakers {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 0.5rem;
+  }
+
+  .see-more {
+    display: none;
+  }
+
+  .partner {
+    width: 100%;
+    padding: 2.2rem 1rem;
+    background: #414246;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 0.5rem;
+  }
+
+  .partner h2 {
+    font-family: 'Lato', sans-serif;
+    font-weight: 400;
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+    color: #fff;
+    padding: 0.5rem;
+    text-align: center;
+  }
+
+  .partner hr {
+    width: 45px;
+    border: 1.5px solid #a04038;
+    margin-bottom: 1rem;
+  }
+
+  .partner-logos {
+    display: flex;
+    justify-content: center;
+    gap: 1rem;
+    flex-wrap: wrap;
+  }
+
+  footer {
+    width: 100%;
+    margin: 0 auto;
+    padding: 4rem 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 10%;
+  }
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180706453), please make sure to add appropriate padding/spacing around the footer elements to match the design guideline. Right now, the `footer` has way a lot of padding 😓Please reduce this ⚒

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235275841-ff8398d8-6417-4d0d-acce-8607cbca9877.png) | ![image](https://user-images.githubusercontent.com/106119105/235275750-15d9280b-8651-4a3b-ac25-a52da71697b5.png)